feat: docs builder documentation (#395)
vtexdocs/dev-portal-content: docs/vtex-io/Reference/concepts/vtex-io-documentation-builders.md

 `admin` | Exports blocks and routes to the VTEX Admin.| ❌ |
 `assets` | Handles assets used by store theme blocks. It gets all asset paths used and uploads them to the VTEX IO database. *You can learn how to use it by accessing the recipe on [Using the Assets Builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-using-the-assets-builder/).*| ❌ |
 `configuration` | Allows you to allocate code pertaining to service configurations to an app that's independent of the platform. *You can learn how to use it by accessing the recipe on [Developing service configuration apps](https://developers.vtex.com/docs/guides/vtex-io-documentation-developing-service-configuration-apps/).*| ❌ |

>  `docs` | Allows developers to add documentation in the `/docs` directory, to be published in our Developer Portal. *You can learn how to use it by accessing the recipe on [Using the Docs Builder to publish app documentation](https://developers.vtex.com/docs/guides/using-the-docs-builder-to-publish-app-documentation/).*| ❌ |

Reviewer: ```suggestion
 `docs` | Allows developers to publish documentation from the `/docs` directory in our Developer Portal. *You can learn how to use it by accessing the recipe on [Using the Docs Builder to publish app documentation](https://developers.vtex.com/docs/guides/using-the-docs-builder-to-publish-app-documentation/).*| ❌ |
```
